tools/create-extension/_README.md

```diff
@@ -7,9 +7,8 @@ A WooCommerce Admin Extension
 To get started, run the following commands:
 
 ```text
-pnpm install
-pnpm start
+npm install
```

Comment:
This is because the wp script `check-licenses` is not compatible with pnpm. We could also remove reference to that script instead. 

Maybe at some point we should upstream an improvement to wp-scripts to support pnpm in this case, but I'm not sure if they'll want that.